src/org/myworldgis/netlogo/VectorFeature.java

```diff
@@ -252,6 +252,14 @@ public Envelope getEnvelope () {
         return _geometry.getEnvelopeInternal();
     }
 
+    /** */
+    public double getDistance (Geometry geometry) {
```

Comment:
Need to return a `Double` here as it can be `null` and so the value type of `double` won't work.